wp-content/themes/twentyseventeen-wp20/style.css

```diff
@@ -903,6 +903,7 @@ body.page-slug-whats-on h1.entry-title {
 #wp20-events-map {
 	background-color: #AADAFF;
 	margin: 0 auto 40px;
+	height: 508px;
```

Comment:
Figma shows this as 490px high on mobile and 530px high on desktop

| Mobile | Desktop |
|-|-|
| ![Screen Shot 2023-04-06 at 2 37 05 PM](https://user-images.githubusercontent.com/1017872/230258546-1784e62e-0681-4e77-addf-321df9de36d4.jpg) | ![Screen Shot 2023-04-06 at 2 37 34 PM](https://user-images.githubusercontent.com/1017872/230258565-a088ad42-a5c0-45ac-ba17-770c3f3842ae.jpg) |

Reply:
Huh, you're right. I remember seeing `508px` somewhere, but I don't see it now. 

Fixed in 7868dd1, thanks!